src/pages/yujinLee/Login/Login.js

```diff
@@ -1,6 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+// import { Link } from "react-router-dom";
+import './Login.scss';
+
 const Login = () => {
-  return <h1>Login입니다.</h1>;
+  const navigate = useNavigate();
+
+  const [userId, setUserId] = useState('');
+  const [userPw, setUserPw] = useState('');
+
+  const saveUserId = e => {
+    setUserId(e.target.value);
+  };
+
+  const saveUserPw = e => {
+    setUserPw(e.target.value);
+  };
+
+  return (
+    <div className="loginWrapper">
+      <div className="loginBox">
+        <div className="logo">westagram</div>
+        <form className="Container">
+          <input
+            onChange={saveUserId}
+            type="text"
+            className="loginId"
+            name="userID"
+            minLength="5"
+            maxLength="20"
+            autoFocus="turu"
+            autoComplete="off"
+            placeholder="전화번호, 사용자 이름 또는 이메일"
+          />
+          <input
+            onChange={saveUserPw}
+            type="password"
+            className="loginPw"
+            name="userPW"
+            maxLength="11"
+            autoComplete="new-password"
+            placeholder="비밀번호"
+          />
+          {/* <Link to="/main"> */}
+          <button
+            onClick={() => {
+              navigate('/main');
+            }}
+            type="submit"
```

Comment:
button 태그의 default type이 submit이므로 명시할 필요 X
```suggestion
            type="button"
```
위와 같이 명시하면 form 태그의 제출하는 기본 동작을 수행하지 않을 수 있음